### DISCUSSION TITLE
Problems when driving a car with 2.5D view

### DISCUSSION THREAD

**Discussion by sjvudp** [osmandapp/OsmAnd#18081] [General]
I have roughly three issues when driving a car with 2.5D mode in landscape orientation on Android:

1. Obviously the 2.5D mode only makes sense when using either compass orientation or movement orientation
2. Dynamic zoom is zooming out too early (i.e. at rather low speeds), so you miss nearby details.
3. The position of the virtual car seems to be at the upper third of the screen, when it actually should be at the lower third of the screen (allowing you too see more of _where you are going to_)

The screenshot illustrates at least two of the three issues (decluttering the screen would be another useful feature)
![Screenshot_2023-09-02-22-41-56-06_54166e5c6f8ad9491c65e2397c495c92](https://github.com/osmandapp/OsmAnd/assets/6734808/4f75c5e3-f0af-493f-9b8b-1c2ddb388599)

**Comment by DmitryAlexei** [comment]
Some answers to your questions you can find in documentation https://osmand.net/docs/user/navigation/guidance/map-during-navigation

**Reply by sjvudp** [reply]
My settings were like "center after 10 seconds" and "zoom to close-up" and "snap to roads". So the recommended "Mid-range" should even be worse.

**Comment by DmitryAlexei** [comment]
For display position see https://osmand.net/docs/user/personal/profiles#general-settings

**Reply by sjvudp** [reply]
My setting was "automatic".

**Comment by DmitryAlexei** [comment]
Please try the beta from Google play (or upcoming release 4.6) and see if it is good for you. 
If your issue persists, please, export your profile setting and attach it to the comment (menu > settings > your profile > swipe to the bottom > export profile )
